tests/unit_test_pipeline.py

```diff
@@ -0,0 +1,18 @@
+from steps import prepare_data
+from zenml import pipeline
+
+
+@pipeline
+def {{PLACEHOLDER}}_full_finetune(
+    system_prompt: str,
+    base_model_id: str,
+    use_fast: bool = True,
+    load_in_8bit: bool = False,
+    load_in_4bit: bool = False,
+):
+    """Very simplified pipeline for unit testing only."""
+    datasets_dir = prepare_data(
```

Comment:
This is still vulnerable to what we were talking about last week w/r/t the hugging face service needing to be up and running for our tests to run. Would it be hard to extract out a subset of the data and to mock the call to load the data from hugging face for the purposes of this test?

Reply:
Well, if I mock more it doesn't make sense to run this at all, IMO. Maybe just check the building of the template and import stuff from it, just validating that the flow is somewhat working - not a silver bullet, but better than just running fully mocked code 🙂